include/picongpu/param/radiationObserver.param

```diff
@@ -117,12 +117,12 @@ namespace picongpu
                     const picongpu::float_64 phi(my_index_phi * delta_angle_phi - angle_phi_start);
 
                     /* helper functions for efficient trigonometric calculations */
-                    picongpu::float_32 sinPhi;
-                    picongpu::float_32 cosPhi;
-                    picongpu::float_32 sinTheta;
-                    picongpu::float_32 cosTheta;
-                    pmacc::math::sincos(precisionCast<picongpu::float_32>(phi), sinPhi, cosPhi);
-                    pmacc::math::sincos(precisionCast<picongpu::float_32>(theta), sinTheta, cosTheta);
+                    picongpu::float_X sinPhi;
```

Comment:
I thin this should be 
```suggestion
                    float_X sinPhi;
```
I will fix this asap.

Reply:
Please introduce your own type within `radiation.param` which can be adjusted by the user.